### PR TITLE
feat(DDI-1077): add ability for Dropdown component to render native s…

### PR DESCRIPTION
**Pull request by chrisolsen**
…elect element

### CODE REVIEW

File: libs/docs/src/components/common/Dropdown.stories.mdx [hunk 1]
```diff
@@ -76,6 +76,11 @@ import { useState } from "react";
     type="string"
     description="Defines how the selected value will be translated for the screen reader. If not specified it will fall back to the name"
   />
+  <Prop
+    name="native"
+    type="boolean"
+    description="When true will render the native <select> html element"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Dropdown.stories.mdx:82]: Add `defaultValue="false"` to align with the rest of our boolean values in the table.

File: libs/web-components/src/components/input/Input.svelte [hunk 2]
```diff
@@ -246,10 +251,10 @@
     flex: 1 1 auto;
   }
 
-  input[type=text],
-  input[type=date],
+  input[type="text"],
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/input/Input.svelte:254]: Just a question, not really a comment. Maybe this is auto formatting, but doesn't this potentially change the meaning?

File: libs/docs/src/components/common/Dropdown.stories.mdx [hunk 3]
```diff
@@ -41,18 +41,18 @@ import { useState } from "react";
...
+    description="Maximum height of the dropdown menu items popover. **Non-native only**"
   />
   <Prop
     name="placeholder"
     type="string"
-    description="The text displayed for the dropdown before a selection is made."
+    description="The text displayed for the dropdown before a selection is made. **Non-native only**"
   />
   <Prop
     name="width"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Dropdown.stories.mdx:58]: I noticed this property is not in the NativeSelect. Should we expand its description as well to say `**Non-native only**`?

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 4]
```diff
@@ -217,82 +227,100 @@
...
+    onSelect(option.value, option.label, false);
+  }
 </script>
 
 <!-- Template -->
-
 <div
   data-testid={`${name}-dropdown`}
-  class="goa-dropdown-box"
+  class="dropdown"
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:241]: Could this potentially be considered a breaking change? I'm wondering about people that might have their own javascript, or tests, that maybe target this class.

File: libs/web-components/src/components/native-select/NativeSelect.svelte [hunk 5]
```diff
@@ -0,0 +1,116 @@
+<svelte:options tag="goa-select" />
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/native-select/NativeSelect.svelte:1]: Why do we have this component, if we're also just allowing Dropdown to have a `native` property?

File: libs/web-components/src/components/dropdown/Dropdown.svelte (hunk 4 above)
Comment (chrisolsen) [reply libs/web-components/src/components/dropdown/Dropdown.svelte:241]: No, if that was the case, then any changes we make internally would be considered breaking.

File: libs/web-components/src/components/native-select/NativeSelect.svelte (hunk 5 above)
Comment (chrisolsen) [reply libs/web-components/src/components/native-select/NativeSelect.svelte:1]: Missed deleting file

File: libs/docs/src/components/common/Dropdown.stories.mdx (hunk 3 above)
Comment (chrisolsen) [reply libs/docs/src/components/common/Dropdown.stories.mdx:58]: The `width` attribute is used for both the native and non-native version of the Dropdown

File: libs/web-components/src/components/input/Input.svelte (hunk 2 above)
Comment (chrisolsen) [reply libs/web-components/src/components/input/Input.svelte:254]: It was autoformatting. Meaning is the same

**Pull request merged**